supervisor.conf.j2

```diff
@@ -6,3 +6,8 @@ environment=DJANGO_SETTINGS_MODULE="{{ project_name }}.settings.{{ django_settin
 
 [group:{{ project_name }}]
 programs=gunicorn-{{ project_name }}
```

Comment:
Yeah, I don't think the stack trace above is related to this. Submitted PR to add the Huey process to the group.